website/blog/2024-01-17-running-tabby-as-a-language-server/index.md

```diff
@@ -0,0 +1,59 @@
+---
+slug: running-tabby-as-a-language-server
+title: "Running Tabby as a Language Server"
+authors: []
+tags: []
+---
+
+We are excited to announce that we have added a new feature to Tabby, allowing you to run it as a [language server](https://microsoft.github.io/language-server-protocol/).
```

Comment:
Use first person (`I`) every where, since this is a blog post and you're the author

Reply:
updated.